From 9d5a851460e83a9a56986855d209c142d1678cde Mon Sep 17 00:00:00 2001
From: Jens Almer <bagawk@gmail.com>
Date: Fri, 19 Jun 2026 09:29:48 -0700
Subject: [PATCH] tools/nxp/create_pbl: fix unused variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Makefile is setup to -Werror and newer versions of gcc throw it:
| create_pbl.c: In function ‘add_cpy_cmd’:
| create_pbl.c:616:13: error: variable ‘file_size’ set but not used [-Werror=unused-but-set-variable=]

Upstream-Status: Pending
---
 tools/nxp/create_pbl/create_pbl.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tools/nxp/create_pbl/create_pbl.c b/tools/nxp/create_pbl/create_pbl.c
index c277e391a..8281affab 100644
--- a/tools/nxp/create_pbl/create_pbl.c
+++ b/tools/nxp/create_pbl/create_pbl.c
@@ -613,7 +613,6 @@ int add_cpy_cmd(FILE *fp_rcw_pbi_op)
 	uint32_t ALTCBAR_ADDRESS = BYTE_SWAP_32(0x09570158);
 	uint32_t WAIT_CMD_WRITE_ADDRESS = BYTE_SWAP_32(0x096100c0);
 	uint32_t WAIT_CMD = BYTE_SWAP_32(0x000FFFFF);
-	int file_size;
 	uint32_t pbi_cmd, altcbar;
 	uint8_t pbi_data[MAX_PBI_DATA_LEN_BYTE];
 	uint32_t dst_offset;
@@ -628,7 +627,6 @@ int add_cpy_cmd(FILE *fp_rcw_pbi_op)
 		      pblimg.sec_imgnm);
 		goto add_cpy_err;
 	}
-	file_size = get_filesize(pblimg.sec_imgnm);
 	altcbar = 0xfff00000 & altcbar;
 	altcbar = BYTE_SWAP_32(altcbar >> 16);
 	if (fwrite(&ALTCBAR_ADDRESS, sizeof(ALTCBAR_ADDRESS), NUM_MEM_BLOCK,
@@ -679,7 +677,6 @@ int add_cpy_cmd(FILE *fp_rcw_pbi_op)
 			goto add_cpy_err;
 		}
 		dst_offset += MAX_PBI_DATA_LEN_BYTE;
-		file_size -= MAX_PBI_DATA_LEN_BYTE;
 	} while (!feof(fp_img));
 
 	ret = SUCCESS;
-- 
2.54.0

